config/locales/en.yml

```diff
@@ -17,3 +19,8 @@ en:
     goals: 'Goals View'
     tasks: 'Tasks View'
     profile: 'Profile View'
+  alerts:
+    login:
+      succesfull: 'Login Succesfull'
+      failed: 'Login Failed'
```

Comment:
Let's remove all quotes from this file.
Please run 
```
i18n-tasks normalize
```

Reply:
nice thank you